kontonummer.js

```diff
@@ -348,7 +348,8 @@
             account  : 10,
             control  : 10
         },
-        zerofill: true
+        zerofill: true,
+        warnOnBadChecksum: true, // Special swedbank accounts can cause checksum errors which we conditionally want to suppress
```

Comment:
While we may not have to care about [<IE9 support](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas#Browser_compatibility), this is a public repo that doesn't have any build step and seemingly trailing commas have been avoided, likely for compatibility reasons. I'd suggest to avoid trailing commas in this repo. This applies to a few other places in this PR.

```suggestion
        warnOnBadChecksum: true // Special swedbank accounts can cause checksum errors which we conditionally want to suppress
```